pandas/io/sas/sasreader.py

```diff
@@ -29,8 +29,11 @@ def read_sas(filepath_or_buffer, format=None, index=None, encoding=None,
     DataFrame if iterator=False and chunksize=None, else SAS7BDATReader
     or XportReader
     """
-
     if format is None:
+        buffer_error_msg = "If this is a buffer object rather\
+                than a string name, you must specify a format string"
+        if not isinstance(filepath_or_buffer,compat.string_types):
```

Comment:
I think I got this covered. I'll reread the Pandas submission info for how to ensure PEP8 compliance tonight.